allensdk/brain_observatory/ecephys/write_nwb/_schemas.py

```diff
@@ -28,6 +28,11 @@ class Channel(RaisingSchema):
     probe_horizontal_position = Int(required=True)
     manual_structure_id = Int(required=True, allow_none=True)
     manual_structure_acronym = String(required=True, allow_none=True)
+    anterior_posterior_ccf_coordinate = Float(allow_none=True)
+    dorsal_ventral_ccf_coordinate = Float(allow_none=True)
+    left_right_ccf_coordinate = Float(allow_none=True)
+    impedence = Float(required=False, allow_none=True, default=None)
```

Comment:
you set default values in the code that are different than these `None` values. Why not set all the defaults in the schema, so that the executable is only responsible for getting them into NWB?

Reply:
@NileGraddis Really good point, I've made this change. Just to be extra careful, I also re-ran some sample `ECEPHYS_WRITE_NWB_QUEUE` jobs just to check everything was working as expected.